src/core/Pattern_match.ml

```diff
@@ -78,6 +78,18 @@ type taint_trace_item = {
 type taint_trace = taint_trace_item list [@@deriving show, eq]
 type engine_kind = OSS | Pro [@@deriving show, eq]
 
+(* This type is used by postprocessors to report back the validatity
+   of a finding. No_validator is currently also used when no
+   validation has yet occurred, which if that becomes confusing we
+   could adjust that, by adding another state. This corresponds to
+   a identically named type in semgrep_interfaces output types.*)
+type validation_state =
```

Comment:
Could you refer to the type generated by ATD rather than duplicating it here? Not saying you should do this necessarily, definitely follow any existing convention, but I am curious what you think.

Reply:
I could, but others don't seems to. For example I was following engine_kind which doesn't.